src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -1,60 +1,83 @@
 package mate.academy.model;
 
 import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 import jakarta.persistence.JoinColumn;
 import jakarta.persistence.JoinTable;
-import jakarta.persistence.MapsId;
 import jakarta.persistence.OneToMany;
 import jakarta.persistence.OneToOne;
 import jakarta.persistence.Table;
 import java.util.List;
+import java.util.Objects;
 
 @Entity
-@Table (name = "shopping_carts")
+@Table(name = "ShoppingCarts")
 public class ShoppingCart {
     @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @OneToMany
```

Comment:
The `@OneToMany` annotation is used with a `@JoinTable`, which is not typical. Usually, `@OneToMany` is used with `@JoinColumn` or `mappedBy` for bidirectional relationships. Consider reviewing the relationship mapping to ensure it aligns with your database schema and application requirements.